src/cli.ts

```diff
@@ -235,6 +235,11 @@ export function initArgs(argv: string[]): yargs.Argv<RawOptions> {
         "allow app to stay in system tray. If 'start-in-tray' is set as argument, don't show main window on first start",
       choices: ['true', 'false', 'start-in-tray'],
     })
+    .option('tray', {
```

Comment:
I think you meant tray-menu here.

Reply:
As well:

* Let's add something to `cli.test.ts` for this option. Should just be an extra entry between line 133 and 134 (or thereabouts).
* Lat's modify one of the tay menu manual tests in `.github/manual-test` to inect a tray menu and list what should be expected in it's acceptance criteria.